### ISSUE TITLE
Graphics in "Industries we have worked with" do have redundant alt text

### ISSUE THREAD

**Issue by Tajana-Somogyi** [ALASdoo/alasdoo_com_v2#87]
As the industry is written bellow the graphic, the alt text should be empty.

**Comment by Tajana-Somogyi** [comment]
@vsomogyi Why I cannot see empty alt text in code?
